# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditorEventListener.h
# Commit: b9ebe58001d7
# Full Hash: b9ebe58001d787bb0bbad6d39ae8681966d5a77f
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2020-04-04 09:29:13
# Regressor Bug: 1619914
# File Overlap Count: 1
# Description:
#   Bug 1619914 - part 2: Mark transaction class methods and their caller methods as `MOZ_CAN_RUN_SCRIPT r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69303
# ==============================================================================

diff -r ce2087ddab5c -r b9ebe58001d7 editor/libeditor/EditorEventListener.h
--- a/editor/libeditor/EditorEventListener.h	Fri Apr 03 08:30:37 2020 +0000
+++ b/editor/libeditor/EditorEventListener.h	Fri Apr 03 08:32:01 2020 +0000
@@ -75,7 +75,7 @@
   }
   MOZ_CAN_RUN_SCRIPT virtual nsresult MouseClick(
       WidgetMouseEvent* aMouseClickEvent);
-  nsresult Focus(InternalFocusEvent* aFocusEvent);
+  MOZ_CAN_RUN_SCRIPT nsresult Focus(InternalFocusEvent* aFocusEvent);
   nsresult Blur(InternalFocusEvent* aBlurEvent);
   MOZ_CAN_RUN_SCRIPT nsresult DragEnter(dom::DragEvent* aDragEvent);
   MOZ_CAN_RUN_SCRIPT nsresult DragOverOrDrop(dom::DragEvent* aDragEvent);